ROADMAP.md

```diff
@@ -0,0 +1,50 @@
+# Roadmap
```

Comment:
I'm not sure this is an actual roadmap. A roadmap typically includes a sequential list of logical steps that, together, will achieve the goals outlined in the project's architecture. Ideally, the steps should be somewhat actionable, although implementation details are reserved for individual issues.

Reply:
To avoid redundancy I'd prefer to defer the work package hierarchy to a more suitable (since dynamic) system.

We'll be able to choose from:
- Issues
- Milestones
- Projects

That should satisfy what this could be seen as a prelude for. Still, you'll click on "Roadmap" in the end, even if you'll have to afford one more hop.

Reply:
I was looking at Grafana's `ROADMAP.md` and it basically said "Go look at Github issues and milestones."

Reply:
Yeah, similarily TxPipe, iirc.